tests/verifyRemoteInstances_test.js

```diff
@@ -265,3 +266,37 @@ Data(metrics).Scenario(
     await grafanaAPI.waitForMetric(current.metricName, { type: 'service_name', value: current.serviceName }, 10);
   },
 );
+
+Scenario(
+  'PMM-T1087 Verify adding PostgreSQL remote instance without postgres database @instances',
```

Comment:
I think you did the opposite of what was the expectation for this test case, we have to verify that users can add PostgreSQL remote instances even without having the default Postgres database, what you did is actually just verifying if the user-provided DB exists or not and giving back the error.